yt_dlp/extractor/xvideos.py

```diff
@@ -161,3 +165,95 @@ def _real_extract(self, url):
             'thumbnails': thumbnails,
             'age_limit': 18,
         }
+
+
+class XVideosUserIE(InfoExtractor):
+    _VALID_URL = r'''(?x)
+        https?://(?:.+?\.)?xvideos\.(?:com|es)/
+        (?P<id>(?:channels|amateur-channels|model-channels|pornstar-channels|profiles)/[^/?#&]+)'''
+    _TESTS = [{
+        # channels profile and # in url
+        'url': 'https://www.xvideos.com/channels/college_girls_gone_bad#_tabVideos,videos-best',
+        'info_dict': {
+            'id': 'channels/college_girls_gone_bad',
+            'title': 'College Girls Gone Bad',
+            'description': 'Hot college girls in real sorority hazing acts!',
+        },
+        'playlist_mincount': 109,
+    }, {
+        # model-channels profile and # in url
+        'url': 'https://www.xvideos.com/model-channels/shonariver#_tabVideos,videos-best',
+        'info_dict': {
+            'id': 'model-channels/shonariver',
+            'title': 'Shona River',
+            'description': 'md5:4c695588123b2fb3d39f4d0dbf1da9a0',
+        },
+        'playlist_mincount': 198,
+    }, {
+        # amateur-channels profile
+        'url': 'https://www.xvideos.com/amateur-channels/queanfuckingcucking',
+        'info_dict': {
+            'id': 'amateur-channels/queanfuckingcucking',
+            'title': 'Queanfuckingcucking',
+            'description': 'md5:bf4514baf23bc7d0525727deeaea96a9',
+        },
+        'playlist_mincount': 8,
+    }, {
+        # /profiles/***
+        'url': 'https://www.xvideos.com/profiles/jacobsy',
+        'info_dict': {
+            'id': 'profiles/jacobsy',
+            'title': 'Jacobsy',
+            'description': 'fetishist and bdsm lover...',
+        },
+        'playlist_mincount': 84,
+    }, {
+        # no description
+        'url': 'https://www.xvideos.com/profiles/espoder',
+        'info_dict': {
+            'id': 'profiles/espoder',
+            'title': 'Espoder',
+        },
+        'playlist_mincount': 13,
+    }]
+    _PAGE_SIZE = 36
+
+    def _entries(self, json):
+        for video in json.get('videos'):
+            slug = self._search_regex(r'.*/([^/]+)$', video.get('u'), 'slug')
+
+            yield self.url_result(
+                'https://www.xvideos.com/video%s/%s' % (video.get('id'), slug),
+                ie=XVideosIE.ie_key())
```

Comment:
Are these supposed to be fatal or not? You use `.get`, but dont take care to make following steps non-fatal